hooks/post-command

```diff
@@ -39,15 +44,37 @@ if [[ "${SHOW_SECRET_SNIPPETS}" == "true" ]]; then
     args+=("--show-secret-snippets")
 fi
 
-output_formats=("human" "json" "sarif")
-if [[ ${output_formats[*]} =~ ${OUTPUT_FORMAT} ]]; then
-    args+=("--format=${OUTPUT_FORMAT}")
+scan_formats=("human" "json" "sarif")
+if [[ ${scan_formats[*]} =~ ${SCAN_FORMAT} ]]; then
+    args+=("--format=${SCAN_FORMAT}")
 else
-    echo "+++ 🚨 Invalid Output Format: ${OUTPUT_FORMAT}"
-    echo "Valid Formats: ${output_formats[*]}"
+    echo "+++ 🚨 Invalid Scan Format: ${SCAN_FORMAT}"
+    echo "Valid Formats: ${scan_formats[*]}"
     exit 1
 fi
 
+# Define valid formats
+valid_file_formats=("human" "json" "sarif" "csv-zip")
+
+# Declare result array
+declare -a result
+
+# Read file output formats into result array
+if plugin_read_list_into_result "BUILDKITE_PLUGIN_WIZ_FILE_OUTPUT_FORMAT"; then
+    for format in "${result[@]}"; do
+        if in_array "$format" "${valid_file_formats[@]}"; then
+            args+=("--output=/scan/result/output,${format}")
```

Comment:
Could it conflict with multiple formats? Might not hurt to be more explicit about formats and include it in the filename.

```suggestion
            args+=("--output=/scan/result/output.${format},${format}")
```

Reply:
@scadu good call, but maybe `args+=("--output=/scan/result/output-${format},${format}")` instead?
Whilst `.json` works, `.human`, `.sarif` and `.csv-zip` might be a bit weird?

Reply:
@tomowatt makes sense!